app/dashboard/models.py

```diff
@@ -2128,6 +2207,21 @@ def get_my_grants(self):
         from grants.models import Grant
         return Grant.objects.filter(Q(admin_profile=self) | Q(team_members__in=[self]) | Q(subscriptions__contributor_profile=self))
 
+    @property
+    def team(self):
+        if not self.is_org:
+            return Profile.objects.none()
+        return Profile.objects.filter(organizations__icontains=self.handle)
+
+    @property
+    def get_org_kudos(self):
+        from kudos.models import Token
```

Comment:
same question as earlier on inline imports

Reply:
cant change this one. results in circular import if i do